modules/ztp-worker-node-preparing-policies.adoc

```diff
@@ -0,0 +1,131 @@
+              user:
+                name: root
+            - contents:
+                source: data:text/plain;charset=utf-8;base64,ewogICJtYW5hZ2VtZW50IjogewogICAgImNwdXNldCI6ICIwLTMiCiAgfQp9Cg==
+              mode: 420
+              overwrite: true
+              path: /etc/kubernetes/openshift-workload-pinning
+              user:
+                name: root
+    - fileName: PerformanceProfile.yaml
+      policyName: "config-policy"
+      metadata:
+        name: openshift-worker-node-performance-profile
+      spec:
+        cpu: <4>
+          isolated: "4-47"
+          reserved: "0-3"
+        hugepages:
+          defaultHugepagesSize: 1G
+          pages:
+            - size: 1G
+              count: 32
+    - fileName: TunedPerformancePatch.yaml
+      policyName: "config-policy"
+      metadata:
+        name: performance-patch-worker
+      spec:
+        profile:
+          - name: performance-patch-worker
+            data: |
+              [main]
+              summary=Configuration changes profile inherited from performance created tuned
+              include=openshift-node-performance-openshift-worker-node-performance-profile
+              [bootloader]
+              cmdline_crash=nohz_full=4-47 <5>
+              [sysctl]
+              kernel.timer_migration=1
+              [scheduler]
+              group.ice-ptp=0:f:10:*:ice-ptp.*
+              [service]
+              service.stalld=start,enable
+              service.chronyd=stop,disable
+        recommend:
+        - profile: performance-patch-worker
+----
+<1> The policies correspond to all clusters with this label.
+<2> The `MCP` field must be set to `worker`.
+<3> This generic `MachineConfig` CR is used to configure workload partitioning on the worker node.
+<4> The `cpu.isolated` and `cpu.reserved` fields must be configured for each particular hardware platform.
+<5> The `cmdline_crash` CPU set must match the `cpu.isolated` set in the `PerformanceProfile` section.
+
++
+A generic `MachineConfig` CR is used to configure workload partitioning on the worker node. You can generate the content of `crio` and `kubelet` configuration files.
+
+. Add the created policy template to the Git repository monitored by the ArgoCD `policies` application.
+
+. Add the policy in the `kustomization.yaml` file.
+
+. Commit the changes in Git, and then push to the Git repository being monitored by the GitOps ZTP ArgoCD application.
+
+[id="ztp-additional-worker-remediate-policies_{context}"]
+== Remediating worker node policies
+
+To remediate the new policies to your spoke cluster, create a new TALM custom resource:
+
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cnfdf15-worker-policies
+  namespace: default
+spec:
+  backup: false
+  clusters:
+  - cnfdf15
+  enable: true
+  managedPolicies:
+  - cnfdf15-worker-config-policy
```

Comment:
```suggestion
  - cnfdf15-workers-config-policy
  - cnfdf15-config-policy
```
OK, so I'm adding here the original cluster config policy, since we are modifying the existing sr-iov configurations, group-du-sno is already there (for modified ptp configuration).
We only add WLP and performance for the worker, as listed above. correct?

Reply:
Based on other parts of the doc, I think the policies should be: 

managedPolicies
- example-sno-workers-config-policy  # for WLP, ptpconfig, and performanceprofile 
- example-sno-workers-sriov-config-policy  # for sriovnetworks, pollicies, etc. sriov cannot use the same policy because it has different wave number in ZTP source-cr than ptpconfigs and performanceprofile/tuned, etc.
- group-du-sno  # sriov/ptp operator configs modified in existing group PGT. We need to add this because after the modification, this policy is now NonCompliant, and need to be enforced via CGU.

Reply:
Not sure why do we need example-sno-workers-sriov-config-policy.
Before worker addition we had 3 policies:
- example-sno-config-policiy - for sriov
- common-example-sno-subscriptions-policy - for subscriptions
- group-du-sno-config-policy - for ptp configs etc.

Now we add example-sno-workers-config-policy - for WLP and performance profile
PTP config and SR-IOV config we change to select worker instead of master without adding new policies. 
example-sno-config-policiy and group-du-sno-config-policy become non-compliant.  The example-sno-workers-config-policy is non-compliant since it is new. so managedPolices should contain
- example-sno-workers-config-policy
- group-du-sno-config-policy 
- example-sno-config-policiy

Correct?

Reply:
I thought to avoid downtime, you were suggesting to add new policy using worker role to new worker config. 

I'm not sure about the underlying differences between modifying existing config policy from master to worker, vs adding new policy with worker using same config. Perhaps I misunderstood you - maybe you were comparing worker role vs a new role? If so, then let's just follow what you suggested above.